.changeset/fuzzy-lemons-cough.md

```diff
@@ -0,0 +1,8 @@
+---
+"ledger-live-desktop": minor
```

Comment:
Do we still need to split this into multiple changesets since we now deliver both at once ?

Reply:
Nope